packages/cli/src/commands/web/page/create.ts

```diff
@@ -16,6 +18,30 @@ const command: GluegunCommand = {
       if (result && result.name) name = result.name;
     }
 
+    const types = await prompt.ask({
+      type: "multiselect",
+      name: "types",
+      message: "What types of page do you want to create?",
+      choices: [
+        { name: "page", message: "Page" },
+        { name: "pageLoad", message: "Page Load" },
+        { name: "pageServer", message: "Page Server" },
+      ],
+    });
+
+    const ejsMap = {
+      page: "page.ejs",
+      pageLoad: "page-load.ejs",
+      pageServer: "page-server.ejs",
+    };
+
+    console.log(ejsMap);
+
+    if (types.types.length < 1) {
```

Comment:
maybe we should map the nested `types` to the top. Couldn't think of a good variable name for this though lol